interfaces/Portalicious/src/app/components/query-table/query-table.component.html

```diff
@@ -287,19 +287,22 @@
       class="hover:bg-white"
     >
       <td [attr.colspan]="totalColumnCount()">
-        <div class="w-full py-6 text-center">
+        <div class="w-full py-6 text-left">
```

Comment:
`text-start` for RTL-support.

Or isn't the "default text alignment" not good already? (or is some parent-element already defining `-center`?

Reply:
When will I leeeeaaaaaaarn Elwin lol

Reply:
There must be an ESLint-rule for this... ;)

Reply:
Not yet but we're not alone 😄 https://github.com/francoismassart/eslint-plugin-tailwindcss/issues/235